ui/src/pages/insightDetail/components/eventAggregator/styles.module.less

```diff
@@ -349,13 +346,12 @@
         }
       }
 
-      .events_content_diagnosisContent {
+      .events_content_diagnosisBody {
+        position: relative;
         overflow-y: auto;
```

Comment:
The new `border-radius: 0 0 12px 12px;` significantly changes the visual design. Ensure this matches the intended design.